src/python_routing_v02/troute/routing/compute.py

```diff
@@ -899,7 +900,18 @@ def compute_diffusive_routing(
                     trib_flow = np.append(trib_flow, i[1][x, ::3], axis = 0)  
                     
         # create DataFrame of junction inflow data            
-        junction_inflows = pd.DataFrame(data = trib_flow, index = trib_segs)
+        junction_inflows = pd.DataFrame(data = trib_flow, index = trib_segs)        
+     
+        # very very temporary installment for natual channel x-section data
+        inland_bathyNC = nc.Dataset('/glade/work/donghakim/cape_fear/bathy/eHydro_ned_cross_sections_2021_12_29.nc')
```

Comment:
Any data I/O should happen in python_framework_v02/troute/nhd_io.py. I prefer that we make a new function to read-in bathy data from netCDF and format it as needed. Then, these data would need to get passed through the workflows (_main_.py) as an input to compute.py